core/src/main/scala/replpp/Operators.scala

```diff
@@ -0,0 +1,121 @@
+package replpp
+
+import replpp.shaded.os.{ProcessInput, ProcessOutput, SubProcess}
+
+import java.io.FileWriter
+import java.lang.ProcessBuilder.Redirect
+import java.lang.System.lineSeparator
+import java.nio.file.{Files, Path, Paths}
+import scala.jdk.CollectionConverters.IterableHasAsScala
+import scala.sys.process.Process
+import scala.util.{Try, Using}
+
+/**
+  * Operators to redirect output to files or pipe them into external commands / processes,
+  * inspired by unix shell redirection and pipe operators: `>`, `>>` and `|`.
+  * Naming convention: similar to scala.sys.process we prefix all operators with `#`
+  * to avoid naming clashes with more basic operators like `>` for greater-than-comparisons.
+  * */
+object Operators {
+
+  extension (value: String) {
+
+    /** Redirect output into file, overriding that file - similar to `>` redirection in unix. */
+    def #>(outFile: Path): Unit =
+      Files.writeString(outFile, value)
```

Comment:
for symmetry with append-mode, I'd also append an empty line here

Reply:
thanks, just refactored for reuse and symmetry